templates/interface/filter-panels.interface.hbs

```diff
@@ -51,21 +51,83 @@
               <option value=">=">Greater or equal to</option>
               <option value="<">Less than</option>
               <option value="<=">Less or equal to</option>
+              <option value="between">Is between</option>
             </select>
             <span class="icon fa fa-chevron-down"></span>
             <span class="select-value-proxy">-- Please wait</span>
           </label>
         </div>
       </div>
 
-      <div class="form-group clearfix" id="filter-value">
+      <div class="form-group clearfix" id="filter-value-type-{{id}}">
+        <div class="col-sm-4 control-label">
+          <label>Value type</label>
+        </div>
+        <div class="col-sm-8">
+          <label for="value-type-field-{{id}}" class="select-proxy-display">
+            <select name="value-type-field-{{id}}" id="value-type-field-{{id}}" data-field="valueType" class="hidden-select form-control">
+              <option value="enter-value">Enter a value</option>
+            </select>
+            <span class="icon fa fa-chevron-down"></span>
+          </label>
+        </div>
+      </div>
+
+      <div class="form-group clearfix" id="filter-value-{{id}}">
         <div class="col-sm-4 control-label">
           <label for="value-field-{{id}}">Value</label>
         </div>
         <div class="col-sm-8">
           <input type="text" name="value-field-{{id}}" id="value-field-{{id}}" data-field="value" class="form-control">
         </div>
       </div>
+    </div>
+
+    <div id="logic-comparison-{{id}}">
+      <div class="form-group clearfix" id="filter-value-type-from-{{id}}">
```

Comment:
What's this ID used for? It seems unnecessary as I don't see it being referred to in CSS or JS.

```suggestion
      <div class="form-group clearfix">
```

Reply:
I believe that after resolving conflicts, I lost part of the code, I will now double-check the history, sorry

Reply:
@tonytlwu, yes you're right ids is not used, sorry, already removed them